docs/src/introduction/overview.md

```diff
@@ -85,7 +85,6 @@ license = "Apache-2.0"
 name = "wallet_contract"
 
 [dependencies]
```

Comment:
We should probably remove `[dependencies]` to match the behaviour of `forc init`.

Reply:
`forc init` has a `[dependencies]`, as does `cargo init`.

Reply:
What's actually bad is that we actually discussed this in my PR https://github.com/FuelLabs/sway/pull/1226 10 days ago.
![image](https://user-images.githubusercontent.com/59666792/164677098-a050a833-c2a2-4e95-97d7-b6cdb34a4f34.png)